src/js/HeartRateSearch.js

```diff
@@ -18,32 +20,70 @@ const muiThemeWhiteText = getMuiTheme({
 });
 
 
-
 class HeartRateSearch extends Component {
 
   constructor() {
     super();
     this.state = {
       currUser: null,
       userHR: [null],
+      numHR: null,
+      maxHR: null,
       graphVisible: false,
     };
     this.getHR = this.getHR.bind(this);
```

Comment:
if you don't want to use these `bind`s what you can do is have your class functions be arrow functions (which will automatically bind `this`). So for example for `getHR(event)` you could instead make that:

```js
class HeartRateSearch extends Component {
...
    getHR = event => {
        ...
    }
}
```
And you would not have to call the `bind`s in the constructor anymore :).